version/version.go

```diff
@@ -1,5 +1,28 @@
 package version
 
+import (
+	"fmt"
+	"runtime"
+
+	sdkVersion "github.com/operator-framework/operator-sdk/version"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+var log = logf.Log.WithName("version")
+
 var (
-	Version = "0.0.1"
+	Version   = "" // version will be replaced while building the binary using ldflags
```

Comment:
Do we need to expose Version if we are exposing a Get() for it?

Reply:
cc. https://github.com/openshift/windows-machine-config-operator/pull/82#discussion_r456053518

Reply:
Yes, because we are setting `Version` using ldflags, we need to expose it.